### PR TITLE
Clarifying vulnerability payment amounts and categories

### PR DESCRIPTION
**Pull request by chayim**
This pull request clarifies our vulnerability payment amounts, based on category, in descending order of payment amount.

My approach to outcomes stems from the relative value of the vulnerability from our point of view. For example, a user reading or writing to another user's code, encompasses the variety of data loss possibilities, as well as modifying customer code. That should be the holy grail.

We still want to encourage reports on misconfigurations that could lead to a security event; the early we know about a possibility, the earlier we can address it . As such that is the single, non-user focused outcome in this list.

### CODE REVIEW

File: handbook/engineering/security/reporting-vulnerabilities.md [hunk 1]
```diff
@@ -4,13 +4,23 @@ If you think that you have found a security or privacy vulnerability, please ema
 
 ### Bounties
 
-We provide monetary rewards, from \$50 to \$10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.
+We provide monetary rewards, from $50 to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.
```
Comment (ElizabethStirling) [inline handbook/engineering/security/reporting-vulnerabilities.md:7]:
```suggestion
We provide monetary rewards, from $50 to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit, and the quality of the security report.
```

**Review by ElizabethStirling — APPROVED** [review]

File: handbook/engineering/security/reporting-vulnerabilities.md [hunk 2]
```diff
@@ -4,13 +4,23 @@ If you think that you have found a security or privacy vulnerability, please ema
...
+When a monetary bounty is presented, eligible reports will be based on the severity, as determined by [CVSS v3.1](https://www.first.org/cvss/calculator/3.1). We will send payment to a valid PayPal account. We will ask you for the name and country associated with your PayPal account.
+
+**Categories**
+
+|Attack Outcome| Maximum Payout|
+|:------------- | :----------: |
+| You read or write to another user's code | $10,000 |
+| You take over another user's account  | $5000 |
+| You gain access to the internal Sourcegraph cloud network | $2500 |
+| You gain access to another user's configurations | $2000 |
```
Comment (nicksnyder) [inline handbook/engineering/security/reporting-vulnerabilities.md:18]: Consistently use comma (or not)

```suggestion
| You take over another user's account  | $5,000 |
| You gain access to the internal Sourcegraph cloud network | $2,500 |
| You gain access to another user's configurations | $2,000 |
```

File: handbook/engineering/security/reporting-vulnerabilities.md [hunk 3]
```diff
@@ -4,13 +4,23 @@ If you think that you have found a security or privacy vulnerability, please ema
...
+We provide monetary rewards, from $50 to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit, and the quality of the security report.
 
-We will send payment to a valid PayPal account. We will ask you for the name and country associated with your PayPal account.
+When a monetary bounty is presented, eligible reports will be based on the severity, as determined by [CVSS v3.1](https://www.first.org/cvss/calculator/3.1). We will send payment to a valid PayPal account. We will ask you for the name and country associated with your PayPal account.
+
+**Categories**
+
+|Attack Outcome| Maximum Payout|
```
Comment (nicksnyder) [inline handbook/engineering/security/reporting-vulnerabilities.md:13]:
```suggestion
| Attack Outcome | Maximum Payout |
```

**Review by nicksnyder — APPROVED** [review]
Clear and concise, thanks!

**Pull request merged**